geomesa-raster/src/test/scala/org/locationtech/geomesa/raster/RasterTestsUtils.scala

```diff
@@ -54,6 +54,19 @@ object RasterTestsUtils {
     new RasterQuery(bb, res, None, None)
   }
 
+  implicit def bboxToRefEnv(b: BoundingBox): ReferencedEnvelope = new ReferencedEnvelope(b.minLon,
```

Comment:
Long lines.  Move the end of the line, 'new ReferencedEnvelope...' to the next line.